README.md

```diff
@@ -1,6 +1,9 @@
 # Bitstyles
 
-A developing collection of CSS styles and helpers, for use in Bitcrowd projects.
+A collection of CSS objects and helpers. See the [Bitcrowd frontend styleguide](https://github.com/bitcrowd/frontend) for details of the principles on which this collection is built. The TL;DR: it’s a variation of the BEMATOMICOOCSS methodology:
+- BEM naming convention
+- Atomic CSS-inspired, the object classes aim to provide most functionality needed to create whichever layout or UI you need. Trump classes create the exceptions that inevitably occur.
+- When multiple object classes on an element just don’t make much sense any more (too many classes becomes unreadable), or the element is a repeated and consistent part of your UI, all the object classes are provided as Sass mixins for you to create your own components.
```

Comment:
There’s an implied (read: missing) word here, that parenthetical should perhaps be be 'using too many classes becomes unreadable'. I’ll read over it again